docs/developer/introduction.md

```diff
@@ -115,7 +115,7 @@ describe. For example, take a look at [api/src/](https://github.com/openkfw/TruB
 [githubdiscussions]: https://github.com/openkfw/TruBudget/discussions
 [architecture decision records (adr)]: http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions
 [doc/architecture/decisions]: ./architecture/intro.md
-[developer setup]: https://github.com/openkfw/TruBudget/blob/master/doc/installation/Developer-Setup.md#developer-setup/
+[developer setup]: https://github.com/openkfw/TruBudget/blob/master/docs/developer/developer-setup.md#developer-setup/
```

Comment:
**DO NOT** use absolut path to GitHub. Use relativ link. This will break the documentation!

Reply:
done

Reply:
no. most of the direct links point to a readme in a sub project this is fine. this link points to the doc folder. pls use relative link instead.